app/assets/javascripts/backbone/views/loan_charts_view.coffee

```diff
@@ -52,18 +58,27 @@ class MS.Views.LoanChartsView extends Backbone.View
     chart.draw(data, options)
 
   breakevenProductProfitChart: ->
-    data = new google.visualization.DataTable();
+    data = new google.visualization.DataTable()
     data.addColumn 'string', I18n.t('loan.breakeven.product')
     data.addColumn 'number', I18n.t('loan.breakeven.profit')
+    options = @defaultChartOptions
 
+    negativeNums = false
```

Comment:
I am not a fan of how the bar chart is visually represented on the page, so I may remove this logic and just let Google Charts display the message it displays when an item is negative.